src/coreComponents/physicsSolvers/fluidFlow/integratedTests/singlePhaseFlow/compressible_1d.xml

```diff
@@ -92,48 +99,18 @@
       referenceViscosity="0.001"
       viscosibility="0.0"/>
 
-    <PoreVolumeCompressibleSolid
-      name="rock"
+    <PressureDependentPorosity
+      name="porosityModel"
       referencePressure="0.0"
+      defaultReferencePorosity ="0.2"
       compressibility="1e-9"/>
+      
+    <ConstantPermeability
+      name="permeabilityModel"
+      permeabilityComponents="1e-12, 1e-12, 1e-12"/>
```

Comment:
These new XML tags look good to me. It would be nice to keep the possibility to import permeabilities and porosities embedded inside external meshes (but I know there is already a lot of work to this PR :) )

Reply:
So, it is certainly possible to use the `FieldSpecification` to set the permeability but I am not sure how to import the field from and external mesh. It is certainly something we need to be able to do though. If not in this PR we should address it right after.